auth0/resource_auth0_attack_protection.go

```diff
@@ -315,67 +332,34 @@ func createAttackProtection(ctx context.Context, d *schema.ResourceData, m inter
 func readAttackProtection(ctx context.Context, d *schema.ResourceData, m interface{}) diag.Diagnostics {
 	api := m.(*management.Management)
 
-	ipThrottling, err := api.AttackProtection.GetSuspiciousIPThrottling()
+	breachedPasswords, err := api.AttackProtection.GetBreachedPasswordDetection()
 	if err != nil {
-		if mErr, ok := err.(management.Error); ok {
-			if mErr.Status() == http.StatusNotFound {
-				d.SetId("")
-				return nil
-			}
-		}
```

Comment:
We're removing the 404 checks as we shouldn't just ignore them. This is not a creatable resource like for example "clients" that when accessing a specific ID, it might not be there. If the endpoint to update breached password detection or any other attack protection types does not exist we should indeed immediately throw back the error as it means we can't manage them at all. Something needs to go terribly wrong for this to happen.